### PR TITLE
gpavelar/c03-docker06

### PR DESCRIPTION
**Pull request by gpavelar**
# Description

Exercise (e.g. C01-E01): C03-DOCKER06

Student Name: Gustavo Avelar.

> Run Docker container passing argument from outside.

## Type of change

- [x] Exercise Submission (one exercise per PR)
  - [x] This PR is just for **one** exercise of a class
  - [x] PR name follows the pattern `<github-username>/<exercise-number>`
  - [x] Added exercise files inside folder `/classes/<class- number>/exercises/<exercise-number>/<github-username>/`
	- [x] Review [exercise submission steps](./README.md#Exercises)
- [ ] Class content (instructors/admins)
- [ ] Documentation update
- [ ] Repo management

### CODE REVIEW

File: classes/03class/exercises/c03-docker06/gpavelar/README.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+
+- [Dockerfile](Dockerfile)
+
+## Command Execution Output
+
+- Commands used to build the image:
+
+```bash
+## Build
+docker build -t exercise06:v1
```
[the remark this answers is not in the document]
Comment (gpavelar) [reply classes/03class/exercises/c03-docker06/gpavelar/README.md:13]: O forgot to add ` . ` at the end of this instructions :/ Sorry

File: classes/03class/exercises/c03-docker06/gpavelar/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,3 @@
+FROM alpine:latest
+FROM gpavelar/exercise02:v1
```
[the remark this answers is not in the document]
Comment (gpavelar) [reply classes/03class/exercises/c03-docker06/gpavelar/Dockerfile]: I just removed the second from. I think i have messed the concepts. I would like to use just the last one, that is also base on alpine. I think, now I followed all required rules. I update also the alpine to use the version `3.7` as you recommended. Thank you for your tips

File: classes/03class/exercises/c03-docker06/gpavelar/Dockerfile [hunk 3]
```diff
@@ -0,0 +1,2 @@
+FROM alpine:3.7
```
Comment (alanlima) [inline classes/03class/exercises/c03-docker06/gpavelar/Dockerfile:1]: Great, using specific alpine version 👌

**Review by alanlima — APPROVED** [review]
Nice @gpavelar 
Thanks for the changes, 🤗

**Pull request merged**